_maps/map_files/generic/CentCom.dmm

```diff
@@ -6450,7 +6450,7 @@
 /turf/open/floor/plasteel/grimy,
 /area/centcom/ferry)
 "pO" = (
-/obj/structure/destructible/cult/tome,
+/obj/structure/destructible/cult/tome/library,
```

Comment:
Why do this in CC?

Reply:
Well you are right it does not really need the change i guess i can just revert that one back to the original object path